jmclient/jmclient/blockchaininterface.py

```diff
@@ -658,7 +506,7 @@ def sync_unspent(self, wallet):
             self._add_unspent_utxo(wallet, u)
         et = time.time()
         log.debug('bitcoind sync_unspent took ' + str((et - st)) + 'sec')
-        self.wallet_synced = True
+        #self.wallet_synced = True
```

Comment:
is there still any good reason to keep the `wallet_synced` variable or can it be factored out completely?

Reply:
Maybe, maybe not. For now the `sync_wallet` function in the `WalletService` still uses it to decide if it succeeded or not. As you doubtless remember that setting has a number of possible places to be set within the blockchaininterface syncing code. I do agree that it'd be nice not to have it be a separate setting at bci level.